spec/mocks/MockCompiler.coffee

```diff
@@ -0,0 +1,57 @@
+$ = require('jquery')
+Backbone = require('backbone')
+
+MockCompiler = Backbone.Model.extend(
+  load: (templateName, onCompletion) ->
+    html = @getHTMLForTemplateName(templateName)
+    onCompletion(html)
+
+  appendToContainer: (container, template) ->
+    $(container).html(template)
+
+  getHTMLForTemplateName: (templateName) ->
```

Comment:
Yeah, at the moment this is a less than ideal solution. It was a quick fix to avoid importing my partials using AJAX, because making an asynchronous AJAX request, even for files in my own directory was causing inconsistent test failures.